internal/service/project/model_project_test.go

```diff
@@ -266,6 +268,7 @@ func TestProjectDataSourceSDKToDataSourceTFModel(t *testing.T) {
 				Limits:                                      limitsTF,
 				IPAddresses:                                 ipAddressesTF,
 				Created:                                     types.StringValue("0001-01-01T00:00:00Z"),
+				Tags:                                        emptyTfTags(),
```

Comment:
these model tests should not have changed IMO - unless customers are adding tests, there shouldn't be a need to specify empty tags. If the API is returning an empty set even when not required we should ignore that

Reply:
Hmm. I believe they have to change.
Since we have added `Tags` to the structs and we do a full comparison in the assertion, I don't see how we can avoid that.
Or maybe you meant something else? That it should be a `types.MapNull()` ?

Reply:
ok got it, I see it now